config/dev.webpack.config.js

```diff
@@ -4,15 +4,14 @@ const { GitRevisionPlugin } = require('git-revision-webpack-plugin');
 const config = require('@redhat-cloud-services/frontend-components-config');
 
 const webpackProxy = {
-    deployment: process.env.BETA ? 'beta/apps' : 'apps',
     useProxy: true,
     proxyVerbose: true,
-    debug: true,
     env: `${process.env.STAGE ? 'stage' : 'prod'}-${process.env.BETA ? 'beta' : 'stable'}`,
-    appUrl: process.env.BETA ? '/beta/insights/image-builder' : '/insights/image-builder',
 };
 
 const { config: webpackConfig, plugins } = config({
+    deployment: process.env.BETA ? 'beta/apps' : 'apps',
+    appUrl: process.env.BETA ? '/beta/insights/image-builder' : '/insights/image-builder',
```

Comment:
Also, why do we not need to declare `appUrl` in the prod config?

Reply:
Ugh this is a mistake sorry, appURL is proxy sepcific; deployment isn't. Changing this.